Flashcards/task/src/flashcards/Main.java

```diff
@@ -56,12 +56,51 @@ public void resetErrors() {
     private ArrayList<String> logFile;
     private Scanner userScanner;
     private boolean isUpAndRunning;
+    private boolean isImportRequested;
+    private boolean isExportRequested;
+    private String importFileLocation;
+    private String exportFileLocation;
 
     private Main() {
         this.flashcardList = new ArrayList<>();
         this.userScanner = new Scanner(System.in);
         this.logFile = new ArrayList<>();
         this.isUpAndRunning = true;
+        this.isImportRequested = false;
+        this.isExportRequested = false;
+    }
+
+    public static void main(String[] args) {
+        Main main = new Main();
+        main.collectArgs(args);
+
+        if (main.isImportRequested) {
+            main.importCardFile(main.importFileLocation);
+        }
+
+        while (main.isUpAndRunning) {
+            main.runGameByAction();
+        }
+
+        if (main.isExportRequested) {
+            main.exportCardListToFile(main.exportFileLocation);
+        }
+    }
+
+    boolean isImportRequested() {
+        return isImportRequested;
+    }
+
+    private void setImportRequested(boolean value) {
+        isImportRequested = true;
+    }
+
+    private boolean isExportRequested() {
+        return isExportRequested;
+    }
+
+    private void setExportRequested(boolean value) {
```

Comment:
Kijk is goed naar deze setters. Naast dat ze onnodig zijn, doe je niks met de value